pkgs/development/python-modules/cfn-lint/default.nix

```
@@ -60,9 +63,12 @@ buildPythonPackage rec {
     "cfnlint.transform"
   ];
 
+  checkInputs = [ pytestCheckHook mock ];
```

tries to call itself
```suggestion
  checkInputs = [ pytestCheckHook mock ];
  preCheck = ''
    export $out/bin:$PATH
  '';
```